test/blackbox/api/fastrtps_deprecated/PubSubWriter.hpp

```diff
@@ -381,6 +381,8 @@ class PubSubWriter
             eprosima::fastrtps::Domain::removeParticipant(participant_);
             participant_ = nullptr;
         }
+
+        initialized_ = false;
```

Comment:
Yeah.. but since the entire directory is going to be removed, we can leave it for the moment just in case some ci or compilation still compiles both test suites